TestStack.BDDfy/Reporters/ReportModel.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace TestStack.BDDfy.Reporters
+{
+    public class ReportModel
+    {
+        public List<ReportModel.Story> Stories { get; set; }
+
+        public ReportModel()
+        {
+            Stories = new List<Story>();
+        }
+
+        public class Story
```

Comment:
Yes. I wanted to re-use Story, Scenario, etc names, so as to have least effect on existing reports, rather than StoryModel, ScenarioModel. I also wanted to have one self-contained report model, rather than 5 or 6 individual models. It serializes/deserializes well and is hopefully easy for custom reports to use.